scripts/certs-self-signed.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+#
+# Generate self-signed certs for the various TiKV components and clients.
+# self-signed are usefult for testing and cannot be used for production.
+#
+#     mkdir certs
+#     cd certs
+#     cert_dir="$(pwd)"
+#     CERT_PATH="${cert_dir}" ../scripts/certs-generate
+#     cd ..
+#     pushd "$GOPATH/src/github.com/tikv/pd"
+#     ./scripts/run-tls --config "${cert_dir}/pd-certs.toml" &
+#     popd
+#     ./scripts/run-tls --config certs/tikv-certs.toml
+#
+set -euo pipefail
+
+# Useful to create an absolute path for the cert
+# Does not have a trailing slash
+CERT_PATH=${CERT_PATH:-""}
+if [[ -n $CERT_PATH ]] ; then
+	CERT_PATH="${CERT_PATH}/"
+fi
+
+if ! [[ -e openssl.cnf ]] ; then
+    cp "$(find / -name openssl.cnf -print -quit 2>/dev/null)" .
```

Comment:
What if there are multiple "openssl.cnf" files?

Reply:
That is what the `-quit` option is for. I will add a note at the top about how it finds a default openssl.cnf

Reply:
This will cause a lot of warnings on my Mac that it needs to access my contact, etc. I don't think a pre-defined openssl.cnf is required at all. You can just create a template using the scripts.